### PR TITLE
Identifier Relocation

### PR DESCRIPTION
**Pull request by kingthorin**
Remove identifiers from all files names, place them in leadin tables, fix all links.

### CODE REVIEW

**Comment by kingthorin** [comment]
> FILE:Testing_for_APIs.md
> [✖] https://github.com/OWASP/OWASP-Testing-Guide-v5/blob/master/document/4_Web_Application_Security_Testing/4.6_Authorization_Testing/4.6.4_Testing_for_Insecure_Direct_Object_References.md → Status: 404
> [✖] https://github.com/OWASP/OWASP-Testing-Guide-v5/blob/master/document/4_Web_Application_Security_Testing/4.6_Authorization_Testing/4.6.3_Testing_for_Privilege_Escalation.md → Status: 404

This is because of deep linking in a new document, and the names changing in this PR. Once it's moved to it's proper location (in a future PR) these can be made into relative links.

**Comment by kingthorin** [comment]
Ready for review.

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11.6_Testing_for_the_Circumvention_of_Work_Flows.md [hunk 1]
```diff
@@ -43,27 +47,27 @@ An electronic bulletin board system may be designed to ensure that initial posts
...
+[Test Integrity Checks](4.11.3_Test_Integrity_Checks.md)
 
-[Test for Process Timing (WSTG-BUSLOGIC-004)](4.11.4_Test_for_Process_Timing_WSTG-BUSLOGIC-004.md)
+[Test for Process Timing](4.11.4_Test_for_Process_Timing.md)
 
-[Test Number of Times a Function Can be Used Limits (WSTG-BUSLOGIC-005)](4.11.5_Test_Number_of_Times_a_Function_Can_Be_Used_Limits_WSTG-BUSLOGIC-005.md)
+[Test Number of Times a Function Can be Used Limits](4.11.5_Test_Number_of_Times_a_Function_Can_Be_Used_Limits.md)
 
-[Test Defenses Against Application Mis-use (WSTG-BUSLOGIC-007)](4.11.7_Test_Defenses_Against_Application_Mis-use_WSTG-BUSLOGIC-007.md)
+[Test Defenses Against Application Mis-use](4.11.7_Test_Defenses_Against_Application_Mis-use.md)
```
Comment (rejahrehim) [inline document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11.6_Testing_for_the_Circumvention_of_Work_Flows.md]: Mis-use or Misuse?
`[Test Defenses Against Application Misuse](4.11.7_Test_Defenses_Against_Application_Misuse.md)`

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11.7_Test_Defenses_Against_Application_Mis-use.md [hunk 2]
```diff
@@ -1,5 +1,9 @@
 # Test Defenses Against Application Mis-use
```
Comment (rejahrehim) [inline document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11.7_Test_Defenses_Against_Application_Mis-use.md]: In #384 updated to `Misuse` same could be done in filenames too

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/README.md [hunk 3]
```diff
@@ -38,39 +38,39 @@ Every application has a different business process, application specific logic a
...
 
-[4.11.6 Testing for the Circumvention of Work Flows (WSTG-BUSLOGIC-006)](4.11.6_Testing_for_the_Circumvention_of_Work_Flows_WSTG-BUSLOGIC-006.md)
+[4.11.6 Testing for the Circumvention of Work Flows](4.11.6_Testing_for_the_Circumvention_of_Work_Flows.md)
 
 In circumventing workflow and bypassing correct sequence testing, we verify that the application does not allow users to perform actions outside of the “approved/required” business process flow. This is important because without this safeguard in place attackers may be able to bypass or circumvent workflows and “checks” allowing them to prematurely enter or skip “required” sections of the application potentially allowing the action/transaction to be completed without successfully completing the entire business process, leaving the system with incomplete backend tracking information.
 
-[4.11.7 Test Defenses Against Application Mis-use (WSTG-BUSLOGIC-007)](4.11.7_Test_Defenses_Against_Application_Mis-use_WSTG-BUSLOGIC-007.md)
+[4.11.7 Test Defenses Against Application Mis-use](4.11.7_Test_Defenses_Against_Application_Mis-use.md)
```
Comment (rejahrehim) [inline document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/README.md]: `[Test Defenses Against Application Misuse](4.11.7_Test_Defenses_Against_Application_Misuse.md)`

**Comment by kingthorin** [comment]
All good, i knew tneted be a conflict after the ogher PR. Ill sort it out. Tks @rejahrehim

**Comment by kingthorin** [comment]
Conflicts addressed. 2 broken links are the same as before (nothing to worry about).

**Comment by kingthorin** [comment]
https://github.com/OWASP/wstg/commit/5ac22011e32d2a63c0a27ec5884dc0912fc26eac includes all the inbound changes.

I'll fix 4.7.4

**Comment by kingthorin** [comment]
Here's a play by play;

- 4.11.0 added.
- 4.11.7 typo fixed.
- 4.11/README remove a ton of text, leave only links.
- 4.12/README remove leadin text, leave only links.
- 4.1.0 added.
- 4.1.1 added number in title (**missed this one**) (fixed typo that was previously missed).
- 4.1/ERADME remove a ton of text, leave only links, and a few touchups.
- 4.2/README remove leadin text, leave only links.
- 4.3/README remove leadin text, leave only links (added a previously missing link).
- 4.4/README remove leadin text, leave only links.
- 4.5.4 relocate text (**missed this one**)
- 4.5/README remove leadin text, leave only links.
- 4.6/README remove leadin text, leave only links.
- 4.7.1 revise one paragraph.
- 4.7/README remove leadin text, leave only links.
- 4.8.3 relocate text (**missed this one**)
- 4.8.4 relocate text (**missed this one**)
- 4.8.5 relocate text (**missed this one**)
- 4.8 README remove leadin text, leave only links.
- 4/README remove leading text, leave only links.
- README title caps fixes.

**Comment by kingthorin** [comment]
Tks! Sorted!

File: document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11.2_Test_Ability_to_Forge_Requests.md [hunk 4]
```diff
@@ -1,5 +1,9 @@
 # Test Ability to Forge Requests
 
+|id               |
```
Comment (ThunderSon) [inline document/4_Web_Application_Security_Testing/4.11_Business_Logic_Testing/4.11.2_Test_Ability_to_Forge_Requests.md]: Upper case ID

File: document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.1_Testing_for_Session_Management_Schema.md [hunk 5]
```diff
@@ -1,9 +1,13 @@
...
 # Testing for Session Management Schema
 
+|ID           |
+|-------------|
+|WSTG-SESS-001|
+
 ## Summary
 
-One of the core components of any web-based application is the mechanism by which it controls and maintains the state for a user interacting with it. To avoid continuous authentication for each page of a website or service, web applications implement various mechanisms to store and validate credentials for a pre-determined timespan. These mechanisms are known as Session Management.
+In order to avoid continuous authentication for each page of a website or service, web applications implement various mechanisms to store and validate credentials for a pre-determined timespan. These mechanisms are known as Session Management and while they are important in order to increase the ease of use and user-friendliness of the application, they can be exploited by a penetration tester to gain access to a user account, without the need to provide correct credentials.
```
[the remark this answers is not in the document]
Comment (kingthorin) [reply document/4_Web_Application_Security_Testing/4.7_Session_Management_Testing/4.7.1_Testing_for_Session_Management_Schema.md]: Tks

**Comment by kingthorin** [comment]
😊

**Pull request merged**